### ISSUE TITLE
generated text is repeating the training data

### ISSUE THREAD

**Issue by ihavetoomanyquestions** [minimaxir/textgenrnn#32]
I have a relatively small dataset; 10-15 lines of recipes, and about 400 recipes in total. i am trying to train textgenrnn to generate new recipes, and I trained 1000 epochs, with the following config:

num_epochs: 1000
gen_epochs: 10
batch_size: 128
prop_keep: 1.0
new_model: True

model_config:
rnn_layers: 2
rnn_size: 128
rnn_bidirectional: False
max_length: 40
dim_embeddings: 100
word_level: False

However, when I generate the text, with temperature ranging from 0.1 to 1.0, the generated text is just copying the trainign data (sometimes, there is slight differences, but most of the times it is just copy-paste). Is there any parameter I should pay attention to avoid overfitting?

**Comment by minimaxir** [comment]
That's unfortunately a causality of low amounts of training data; it's recommended to have atleast 100,000 character sequences for this reason.

Training for 1000 epochs without a lot of training data will cause overfit. You may want to set `dropout` to `0.1` or `0.2` but that might not help much.

**Comment by vinniyo111** [comment]
Using the notebooks/textgenrnn-train-context-wordlevel.ipynb in the first example, or any example, it does not generate new text. It just copies the sentences. 

```
file_path = "../datasets/reddit_rarepuppers_politics_2000.txt"

textgen.reset()
textgen.train_from_file(file_path,
                        new_model=True,
                        max_length=5,
                        num_epochs=20,
                        gen_epochs=5,
                        word_level=True)
```
